modules/virt-reviewing-top-consumers-projects-vms-nodes.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// * virt/logging_events_monitoring/virt-reviewing-virtualization-overview.adoc
+
+[id="virt-reviewing-top-consumers-projects-vms-nodes_{context}"]
+= Reviewing top consumers for projects, virtual machines, and nodes
+
+In the *Administrator* perspective, you can view the {VirtProductName} page where top consumers of resources are displayed.
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. In the *Administrator* perspective in the {VirtProductName} web console, navigate to *Virtualization -> *Overview*.
+
+. Navigate to the *Top Consumers* cards.
+
+. From a *Top Consumer* card, select the type of resource from the drop-down menu: *CPU*, *Memory*, *Used Filesystem*, *Memory Swap*, *vCPU Wait*, or *Storage Throughput*.
+
+. Select *By Project*, *By VM*, or *By Node*. The top five consumers with the highest resource consumption display.
```

Comment:
There is a dropdown that allows the user to select whether they want to see the top 5 or top 10 consumers.

Reply:
Updated the content.